README.MD

```diff
@@ -0,0 +1,68 @@
+#  Foreman Pulp Plugin
+
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management
+While this plugin is part of the Foreman project it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+
+## Getting Started
+
+The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source. 
+The plugin can also be installed as part of the foreman-installer puppet class with the "--[no-]enable-foreman-proxy-plugin-Pulp" option.
```

Comment:
Pulp -> pulp in the installer option.

Reply:
now fixed this to correct lower case option, missed this when fixing other casing issues.